global_buffer/Makefile

```diff
@@ -1,68 +1,155 @@
 #!/bin/bash
 RTL_ROOT = ./rtl
-TB_ROOT = ./testbench
+TB_ROOT = ./sim
 TOP_MODULE = global_buffer
+TILE_MODULE = glb_tile
+
+XRUN = xrun \
+	   -64bit \
+	   -sv \
+	   -sysv \
+	   -l xrun.log \
+	   -notimingchecks \
+	   -access +r \
```

Comment:
This is redundant since at line 20 you define `+access+rw`

Reply:
typo. thanks